lib/engine/game/g_1880/game.rb

```diff
@@ -167,6 +167,42 @@ class Game < Game::Base
                   },
                   { name: '10', distance: 10, price: 1000, num: 10 }].freeze
 
+        def new_auction_round
+          Engine::Round::Auction.new(self, [
+            G1880::Step::SelectionAuction,
+          ])
+        end
+
+        def new_draft_round
+          Engine::Round::Draft.new(self, [G1880::Step::SimpleDraft], reverse_order: false)
+        end
+
+        def next_round!
+          @round =
+            case @round
+            when Round::Stock
+              @operating_rounds = @phase.operating_rounds
+              reorder_players
+              new_operating_round
+            when Round::Operating
+              if @round.round_num < @operating_rounds
```

Comment:
Just a placeholder for now?

Reply:
Yeah, i took the standard next_round! code and only modified to make the auction and draft rounds work.
this code will change a lot to make it work with 1880 round order/SR rules